DateTime.js

```diff
@@ -416,6 +423,17 @@ var Datetime = createClass({
 		return props;
 	},
 
+	eventBoundaryProps: function () {
```

Comment:
just realized that this whole method is static and should probably just be defined outside the component, where I defined the `eventBoundaryEventTypeHandlers`.

Thoughts?